tealium/debug/TealiumDebugServer.swift

```diff
@@ -92,6 +98,11 @@ class TealiumDebugServer {
         
     }
     
+    func setQueueMax(_ queueMax : Int) {
+        if queueMax > 1  && queueMax < TealiumDebugKey.debugQueueSizeLimit {
+            self.queueMax = queueMax
+        }
```

Comment:
a) Recommend renamming 'debugQueueSizeLimit' to something like 'debugOverrideQueueSizeLimit / debugOverrideQueueSizeCap'

b) If the client assigned number exceeds ours, use ours instead of ignoring their request.